### ISSUE TITLE
--semver flag ends in error on query and install commands (0.6)

### ISSUE THREAD

**Issue by Paden** [DefinitelyTyped/tsd#138]
$ tsd query node -v ">=0.8 <0.10"

-> an error occured!

Invalid major version
TypeError: Invalid major version
    at new SemVer (/usr/local/lib/node_modules/tsd/node_modules/semver/semver.js:305:11)
    at Range.test (/usr/local/lib/node_modules/tsd/node_modules/semver/semver.js:1049:15)
    at Function.satisfies (/usr/local/lib/node_modules/tsd/node_modules/semver/semver.js:1098:16)
    at /usr/local/lib/node_modules/tsd/build/tsd/select/VersionMatcher.js:43:32
    at Array.forEach (native)
    at /usr/local/lib/node_modules/tsd/build/tsd/select/VersionMatcher.js:41:22
    at Array.reduce (native)
    at VersionMatcher.filter (/usr/local/lib/node_modules/tsd/build/tsd/select/VersionMatcher.js:40:33)
    at /usr/local/lib/node_modules/tsd/build/tsd/logic/SelectorQuery.js:31:56
    at tryCatch1 (/usr/local/lib/node_modules/tsd/node_modules/bluebird/js/main/util.js:63:19)

**Comment by blakeembrey** [comment]
Hi @Paden, I just gave it a go and didn't run into any issues. Can you give me any more information?

```
$ tsd query node -v ">=0.8 <0.10"

 - node / node-v0.8.8    
$ tsd --version

>> tsd 0.6.0-beta.5
```

**Comment by richmusser** [comment]
I was getting this error too in 0.6.0-beta.5, but looks like it's fixed in 0.6.0, it was caused by a mismatch in the intMax defined in VersionMatcher.ts versus the SemVer module.

**Comment by Paden** [comment]
0.6.0 did fix this. Thanks, @richmusser!
